### ISSUE TITLE
Feature Request: Sugar for installing new learners

### ISSUE THREAD

**Issue by RaphaelS1** [mlr-org/mlr3learners#82]
It would be nice to have some functions like:

```R
install_mlr3_learner("classif.ranger") -> `Learner`
install_mlr3_learners(c("classif.ranger", "classif.featureless")) -> `list`
```

Which on first run can automatically add the relevant code to install from `drat` and for additional runs can look up the learner id in the registry (#81) and then install the relevant packages. This would make it much easier for users to install a learner without needing to know the package name, and would interface very nicely with a registry.

Thanks!

**Comment by pat-s** [comment]
We could think of adding this kind of smartness into `lrn()`.
Internally, we would then

1. Check if requested learner is available in the learner registry of the current session (i.e. has the learner package already been loaded)
	- If not, check if the learner package is installed at all
       - If the learner package is not yet installed, install it (printing a message that we are doing so)

All of this requires that the users have already access to mlr3learners.drat. This would be another step at the start to check if that condition is true. If not, we prompt the users to take action once and add the repo to their .Rprofile with the provided code in the mlr3learners.drat repo.

With all of the above, users could simply do `lrn(<any learner id>)` and things will just work in 99% of all cases.

**Comment by RaphaelS1** [comment]
That would be amazing

**Comment by adibender** [comment]
> That would be amazing

Seconded! I've just spent ~15 minutes trying to understand how to use some learners in proba (since they moved to separate repos) and I was aware that some learners moved from proba to custom repos. Imagine how frustrating this would be for others.

**Comment by pat-s** [comment]
Yeah, I did not have time yet to do this and not sure when I will have.

Happy to look over a PR that does what was outlined before :)
